src/parser/csql_grammar.y

```diff
@@ -540,6 +540,7 @@ static void pt_value_set_collation_info (PARSER_CONTEXT *parser,
 					 PT_NODE *coll_node);
 static void pt_value_set_monetary (PARSER_CONTEXT *parser, PT_NODE *node,
                    const char *str, const char *txt, DB_CURRENCY type);
+static PT_NODE * pt_make_paren_expr_list (PT_NODE * exp);
```

Comment:
I'm not sure this function should declare in this file.

Reply:
Since it's a static function, I think it's ok to declare a function in csql_grammar.y.